### PR TITLE
py-yajl: new package at 0.3.5

### PR DESCRIPTION
**Pull request by codeandkey**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-ujson/package.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+from spack import *
+
+
+class PyUjson(PythonPackage):
+    """Ultra fast JSON decoder and encoder written in C with Python
+       bindings."""
+
+    homepage = "https://github.com/esnme/ultrajson"
+    url      = "https://files.pythonhosted.org/packages/16/c4/79f3409bc710559015464e5f49b9879430d8f87498ecdc335899732e5377/ujson-1.35.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ujson/package.py]: See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#url

File: var/spack/repos/builtin/packages/py-ujson/package.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    version('1.35', sha256='f66073e5506e91d204ab0c614a148d5aa938bdbf104751be66f8ad7a222f5f86')
+
+    depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ujson/package.py]: Can you remove this package from this PR, or rebase? I just merged your `py-ujson` PR.

Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-ujson/package.py]: My mistake. Didn't see that package made it in here, removed.

File: var/spack/repos/builtin/packages/py-yajl/package.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from shutil import rmtree
+from os import symlink
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-yajl/package.py]: These imports are no longer needed.

**Comment by codeandkey** [comment]
ping @adamjstewart, is there anything that still needs attention?

**Pull request merged**